udata_front/theme/gouvfr/templates/macros/delete.html

```diff
@@ -1,8 +1,8 @@
-{% macro follow_btn(obj, class=None) %}
```

Comment:
I'm confused, was this used before to unfollow?

Reply:
`delete.html` is a new file. This is only me doing a copy/paste without changing the name of the macro 😅